src/entry_handler.cpp

```diff
@@ -38,6 +38,11 @@ using namespace std::literals;
 void
 launch_ui() {
   std::string url = "https://localhost:" + std::to_string(net::map_port(confighttp::PORT_HTTPS));
+
+  if (!config::nvhttp.web_ui_address.empty()) {
+    // Don't append internal TCP port as port is mapped by reverse proxy
+    url = config::nvhttp.web_ui_address;
```

Comment:
You mean via the link handler as used by the tray icon on the host? At least in my case, yes. The main benefit of using the reverse proxy for Sunshine in particularly is so that I can used my own valid SSL cert that's managed on a different machine than the Sunshine host. Having the tray icon open the self-signed site means I need to deal with the SSL warnings each time, which cancels out most of the benefit of using the proxy.

Reply:
Do you think this should be optional? In the event of an internet outage, people won't be able to open ui via the tray icon.

Or fallback to the localhost address if the proxy address is unreachable?

Reply:
I've implemented the latter suggestion as it seems to make the best sense and keep things simpler.